addons/website_forum/views/website_forum.xml

```diff
@@ -686,437 +970,553 @@
+                        <div t-field="message.body" class="o_wforum_readable oe_no_empty"/>
+                    </div>
                 </div>
-            </div>
-            <div class="css_editable_mode_hidden">
-                <form t-attf-id="comment#{ object._name.replace('.','') + '-' + str(object.id) }" class="collapse oe_comment_grey js_website_submit_form"
-                        t-attf-action="/forum/#{slug(forum)}/post/#{slug(object)}/comment" method="POST">
-                    <input type="hidden" name="csrf_token" t-att-value="request.csrf_token()"/>
-                    <input name="post_id" t-att-value="object.id" type="hidden" class="mt8"/>
-                    <textarea name="comment" class="form-control" placeholder="Comment this post..."/>
-                    <button type="submit" class="btn btn-primary mt8">Post</button>
-                </form>
-            </div>
+            </t>
         </div>
     </div>
 </template>
 
 <template id="tag" name="Forum Tags">
     <t t-call="website_forum.header">
-        <h1 class="o_page_header">
-            Tags
-        </h1>
-        <p class="text-muted">
-            A tag is a label that categorizes your question with other,
-            similar questions. Using the right tags makes it easier for
-            others to find and answer your question. (Hover the mouse to follow/unfollow tag(s))
-        </p>
-        <nav t-if="pager_tag_chars">
-            <ul class="pagination mt0 mb0">
-                <li class="page-item disabled"><a class="page-link" aria-label="Filter" title="Filter"><i class="fa fa-filter"></i></a></li>
-                <t t-foreach="pager_tag_chars" t-as="tuple_char">
-                    <li t-attf-class="page-item #{'active' if active_char_tag == tuple_char[1] else ''}"><a t-attf-href="/forum/#{slug(forum)}/tag/#{quote_plus(tuple_char[1])}" class="page-link"><t t-esc="tuple_char[0]"/></a></li>
-                </t>
-            </ul>
+        <nav t-if="pager_tag_chars" class="navbar navbar-light bg-light justify-content-start">
+            <t t-if="len(pager_tag_chars) &lt; 11">
+                <span class="navbar-text mr-3">Show Tags Starting By</span>
+                <ul class="pagination mt0 mb0">
+                    <t t-foreach="pager_tag_chars" t-as="tuple_char">
+                        <t t-if="tuple_char_index &lt; 11">
+                            <li t-attf-class="page-item #{active_char_tag == tuple_char[1] and 'active'}"><a t-attf-href="/forum/#{slug(forum)}/tag/#{quote_plus(tuple_char[1])}" class="page-link"><t t-esc="tuple_char[0]"/></a></li>
+                        </t>
+                    </t>
+                </ul>
+            </t>
+            <div t-else="" class="form-inline" role="toolbar" aria-label="Toolbar with button groups">
+                <label for="filter" class="my-1 mr-2">Show Tags Starting By</label>
+                <select name="filter" class="custom-select" onchange="location = this.value;">
```

Comment:
Should be done in JS file.

Reply:
This is part of the Tags stuff that should be defined.